tests/data/almanack/entropy/add_data.py

```diff
@@ -13,42 +12,20 @@
 - To invoke this script within the development environment, use the following command:
   poetry run python add_data.py
 
-- To unzip the created zip files and view the contents, use the following commands:
-  unzip high_entropy_git.zip -d high_entropy_unzipped
-  unzip low_entropy_git.zip -d low_entropy_unzipped
 """
 
 import pathlib
 import subprocess
-import zipfile
-from pathlib import Path
 
 from add_entropy import add_entropy
 
 
-def zip_git_folder(folder_path: str, output_zip_path: str):
-    """
-    Creates a zip file containing the contents of a folder.
-
-    Args:
-        folder_path (str): The path to the folder to be zipped.
-        output_zip_path (str): The path to the output zip file.
-    """
-    folder_path = Path(folder_path)
-    output_zip_path = Path(output_zip_path)
-
-    with zipfile.ZipFile(output_zip_path, "w") as zipf:
-        for file_path in folder_path.glob("**/*"):
-            if file_path.is_file():
-                zipf.write(file_path, file_path.relative_to(folder_path))
-
-
-def commit_changes(directory: str, message: str):
+def commit_changes(directory: dir, message: str):
```

Comment:
would it make sense to move the entire add_data file into the conftest as fixtures. After changing how my git function works and adding parts to the testing suite, I can see how this would work. Is this what you are suggesting and what are your thoughts on this?